src/ape/api/transactions.py

```diff
@@ -122,6 +133,13 @@ def txn_hash(self) -> HexBytes:
         The calculated hash of the transaction.
         """
 
+    @property
+    def hash(self) -> HexBytes:
```

Comment:
Maybe a 0.9 thing, but happy to re-name this attribute fully to match this kind of use case. don't necessarily need it to match a specific naming convention

Reply:
tx.txn_hash does feel a bit redundant

Reply:
I added a TODO comment for 0.9, i still think the alias is now for now